liquidations/angle/index.ts

```diff
@@ -0,0 +1,102 @@
+import axios from "axios";
+import { gql, request } from "graphql-request";
+import { ethers } from "ethers";
+import { providers } from "../utils/ethers";
+import BigNumber from "bignumber.js";
+import { getPagedGql } from "../utils/gql";
+
+const subgraphUrl = "https://api.thegraph.com/subgraphs/name/picodes/borrow";
+
+const vaultDataQuery = gql`
+query vaultDatas {
+    vaultDatas(where: {isActive: true}) {
+      id
+      collateralAmount
+      vaultManager {
+        agToken
+        collateral
+        collateralFactor
+      }
+      owner
+    }
+  }
+`;
+
+type VaultData = {
+    id: string;
+    collateralAmount: BigNumber;
+    vaultManager: {
+        agToken: string;
+        collateral: string;
+        collateralFactor: BigNumber;
+    }
+};
+
+
+const getVaultData = async () => {
+    const vaultData = (await getPagedGql(subgraphUrl, vaultDataQuery, "vaultDatas")) as VaultData[];
+    return vaultData;
+};
+
+const getPrices = async () => {
```

Comment:
We prefer to use llama price api - https://github.com/DefiLlama/DefiLlama-Adapters/blob/de5be4926982d43e875d164e7f56676d0a644822/liquidations/utils/compound-helpers.ts#L41